File: client/Client/Network/Messages/Session/DenyJoinSessionMessage.cs
namespace LoadRunnerClient.Messages
{
    public class DenyJoinSessionMessage
    {
        public const string TYPE = "DenyJoinSessionMessage";

        public DenyJoinSessionMessage()
        {
        }

        public DenyJoinSessionMessage(string reason)
        {
            _reason = reason;
        }

        private string _reason;

        public string reason
        {
            get => _reason;
            set => _reason = value;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/session/DenyJoinSessionMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.session;

public class DenyJoinSessionMessage {

	public static final String TYPE = "DenyJoinSessionMessage";

	private String reason;

	public DenyJoinSessionMessage() {
	}

	public DenyJoinSessionMessage(String reason) {
		this.reason = reason;
	}

	public String getReason() {
		return this.reason;
	}

	public void setReason(String reason) {
		this.reason = reason;
	}
}
